packages/ibm-products-styles/src/components/PageHeader/_page-header.scss

```diff
@@ -917,6 +917,10 @@ $duration: 1000ms;
     }
   }
 
+  .#{$carbon-prefix}--tabs .#{$carbon-prefix}--tabs__nav-link {
+    margin-inline-end: $spacing-01;
+  }
+
```

Comment:
If this is meant to be a temporary workaround, let’s add a comment linking to the [issue from Core](https://github.com/carbon-design-system/carbon/issues/19942).